File: beecrowd/c/bee1142.c
#include <stdio.h>

int main() {
    int n, i = 0;

    scanf("%d", &n);

    for (int cont = 0; cont < n; cont++) {
        printf("%d %d %d PUM\n", (i+1), (i+2), (i+3));
        i+=4;
    }
    
    return 0;
}

File: beecrowd/go/bee1142.go
package main

import (
	"fmt"
)

func main() {

	var n, v int = 0, 0

	fmt.Scanf("%d", &n)

	for cont := 0; cont < n; cont++ {

		fmt.Printf("%d %d %d PUM\n", (v + 1), (v + 2), (v + 3))
		v += 4
	}
}
